qa-include/plugins/qa-widget-related-qs.php

```diff
@@ -65,8 +65,12 @@ public function output_widget($region, $place, $themeobject, $template, $request
 			$themeobject->output('<ul class="qa-related-q-list">');
 
 			foreach ($questions as $question) {
+				$isASelected = $question['selchildid'] != null ? ' qa-a-count-selected' : '';
```

Comment:
`$isASelected` suggests a boolean variable. Wouldn't it make more sense like `$selectedClass` or something similar?

As a side note, isn't an affirmation easier to read than a negation? E.G.: something like this `isset($question['selchildid']) ? ' qa-a-count-selected' : '';`